gstd/src/common/mod.rs

```diff
@@ -19,6 +19,6 @@
 //! Common modules for each Gear smart contract.
 
 pub mod errors;
-#[cfg(target_arch = "wasm32")]
+#[cfg(feature = "common-handlers")]
```

Comment:
shouln't we support common handlers only for wasm32 arch btw?

Reply:
Yes, we should, but in this case we will not be able to run tests that check the message format. I think tests should be located in `handlers.rs`